libs/ui-components/src/components/Events/EventItem.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Icon, Stack, StackItem } from '@patternfly/react-core';
+import InfoCircleIcon from '@patternfly/react-icons/dist/js/icons/info-circle-icon';
+import { ExclamationTriangleIcon } from '@patternfly/react-icons/dist/js/icons/exclamation-triangle-icon';
+
+import { Event } from '@flightctl/types';
+import { DisplayEvent } from './useEvents';
+
+const NormalEventIcon = () => (
+  <Icon status="info" size="md">
+    <InfoCircleIcon />
+  </Icon>
+);
+
+const WarningEventIcon = () => (
+  <Icon status="warning" size="md">
+    <ExclamationTriangleIcon />
+  </Icon>
+);
+
+const EventItem = ({ event }: { event: DisplayEvent }) => {
+  return (
+    <Stack className="fctl-event" key={event.name}>
```

Comment:
no need for `className` or `key`
```suggestion
    <Stack>
```